### PR TITLE
fix(mux): harden Mux parser and fill coverage gaps from #269

### PR DESCRIPTION
**Pull request by junhoyeo**
## Summary

Follow-up to #269 (Mux client support). Fixes issues found during post-merge review:

- **Token clamping**: Add `.max(0)` to all token fields — Mux was the only parser missing this (all 13 others have it)
- **Cost fallback**: Use Mux's `cost_usd` data as fallback when model pricing is unavailable (matching Cursor/Amp pattern)
- **Wrapped display**: Add Mux + 5 other missing clients (Kimi, Qwen, Roo Code, Kilo, Synthetic) to `client_display_name()` and `client_logo_url()`
- **Edge-case tests**: Add 3 tests for negative tokens, cost summation, and multi-colon model keys
- **Translated READMEs**: Add Mux to all sections in ko, ja, zh-cn READMEs (was completely missing)

### Findings from PR #269 review

| # | Finding | Fix |
|---|---------|-----|
| 1 | Missing `.max(0)` token clamping (PR author incorrectly claimed "only Amp clamps") | Added to all 5 token fields in `mux.rs` |
| 2 | `cost_usd` field marked `#[allow(dead_code)]` — parsed but unused | Now used for `source_cost` computation; dead_code annotation removed |
| 3 | No cost fallback when model pricing unavailable | Added fallback in `lib.rs` using source cost (matching Cursor/Amp) |
| 4 | `wrapped.rs` missing Mux in display name/logo maps | Added Mux + 5 other missing clients |
| 5 | Translated READMEs (ko, ja, zh-cn) missing Mux entirely | Added to all 6 sections per file |

### Tests

- 276 core tests passed (including 3 new edge-case tests)
- 271 CLI unit tests passed
- 65 CLI integration tests passed

---

<a href="https://app.devin.ai/review/junhoyeo/tokscale/pull/274" target="_blank">
  <picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://static.devin.ai/assets/gh-open-in-devin-review-dark.svg?v=1">
    Open with Devin
  </picture>
</a>

---
## Summary by cubic
Hardens Mux parsing and completes UI/docs integration. Tokens are clamped, costs fall back to source values, and Mux now appears correctly in Wrapped and translated READMEs (bullets and Windows path fixed).

- **Bug Fixes**
  - Clamp all Mux token buckets to 0 to prevent negative counts.
  - Use summed cost_usd as a cost fallback when pricing is unavailable (applied across parsers).
  - Support model keys with multiple colons (provider split).
  - Remove dead_code on cost_usd; add tests for negatives, cost sum, and multi-colon keys.
  - Fix Mux Windows data path in translated READMEs (single backslash).

- **New Features**
  - Add Mux, Kimi, Qwen, Roo Code, Kilo, and Synthetic to Wrapped display names and logos.
  - Update ko/ja/zh-cn READMEs to include Mux across sections and restore list bullet prefixes.

<sup>Written for commit a58cd9fda3dbe5552ec759f238aa4b5ce9b987d4. Summary will update on new commits.</sup>

### CODE REVIEW

File: README.ja.md [hunk 1]
```diff
@@ -853,6 +857,7 @@ AIコーディングツールはクロスプラットフォームの場所にセ
 | Qwen CLI | `~/.qwen/` | `%USERPROFILE%\.qwen\` | すべてのプラットフォームで同じパス |
 | Roo Code | `~/.config/Code/User/globalStorage/rooveterinaryinc.roo-cline/tasks/` | `%USERPROFILE%\.config\Code\User\globalStorage\rooveterinaryinc.roo-cline\tasks\` | VS Code globalStorageタスクログ |
 | Kilo | `~/.config/Code/User/globalStorage/kilocode.kilo-code/tasks/` | `%USERPROFILE%\.config\Code\User\globalStorage\kilocode.kilo-code\tasks\` | VS Code globalStorageタスクログ |
+| Mux | `~/.mux/sessions/` | `%USERPROFILE%\.mux\sessions\\` | 全プラットフォームで同じパス |
```
[the remark this answers is not in the document]
Comment (junhoyeo) [reply README.ja.md]: Fixed in a58cd9f — removed the extra trailing backslash in the Mux Windows path across all three translated READMEs.

File: README.ko.md [hunk 2]
```diff
@@ -852,6 +856,7 @@ AI 코딩 도구들은 크로스 플랫폼 위치에 세션 데이터를 저장
 | Qwen CLI | `~/.qwen/` | `%USERPROFILE%\.qwen\` | 모든 플랫폼에서 동일한 경로 |
 | Roo Code | `~/.config/Code/User/globalStorage/rooveterinaryinc.roo-cline/tasks/` | `%USERPROFILE%\.config\Code\User\globalStorage\rooveterinaryinc.roo-cline\tasks\` | VS Code globalStorage 작업 로그 |
 | Kilo | `~/.config/Code/User/globalStorage/kilocode.kilo-code/tasks/` | `%USERPROFILE%\.config\Code\User\globalStorage\kilocode.kilo-code\tasks\` | VS Code globalStorage 작업 로그 |
+| Mux | `~/.mux/sessions/` | `%USERPROFILE%\.mux\sessions\\` | 모든 플랫폼에서 동일한 경로 |
```
[the remark this answers is not in the document]
Comment (junhoyeo) [reply README.ko.md]: Fixed in a58cd9f — removed the extra trailing backslash in the Mux Windows path.

File: README.ko.md [hunk 3]
```diff
@@ -129,7 +130,7 @@ AI 지원 개발 시대에 **토큰은 새로운 에너지**입니다. 토큰은
   - 9가지 테마의 GitHub 스타일 기여 그래프
   - 실시간 필터링 및 정렬
   - 깜빡임 없는 렌더링 (네이티브 Zig 엔진)
-- **멀티 플랫폼 지원** - OpenCode, Claude Code, Codex CLI, Cursor IDE, Gemini CLI, Amp, Droid, OpenClaw, Pi, Kimi CLI, Qwen CLI, Roo Code, Kilo, Synthetic 사용량 통합 추적
+ **멀티 플랫폼 지원** - OpenCode, Claude Code, Codex CLI, Cursor IDE, Gemini CLI, Amp, Droid, OpenClaw, Pi, Kimi CLI, Qwen CLI, Roo Code, Kilo, Mux, Synthetic 사용량 통합 추적
```
[the remark this answers is not in the document]
Comment (junhoyeo) [reply README.ko.md]: Fixed in fe1d2de — restored the `- ` markdown list prefix on line 133.

File: README.zh-cn.md [hunk 4]
```diff
@@ -571,7 +575,7 @@ tokscale sources --json
 - **交互式提示**：悬停查看详细的每日分解
 - **每日分解面板**：点击查看每个来源和模型的详情
 - **年份筛选**：在年份之间导航
-- **来源筛选**：按平台筛选（OpenCode、Claude、Codex、Cursor、Gemini、Amp、Droid、OpenClaw、Pi、Kimi、Qwen、Roo Code、Kilo、Synthetic）
+ **来源筛选**：按平台筛选（OpenCode、Claude、Codex、Cursor、Gemini、Amp、Droid、OpenClaw、Pi、Kimi、Qwen、Roo Code、Kilo、Mux、Synthetic）
```
[the remark this answers is not in the document]
Comment (junhoyeo) [reply README.zh-cn.md]: Fixed in fe1d2de — restored the `- ` prefix on the source filtering line.

File: README.zh-cn.md [hunk 5]
```diff
@@ -853,6 +857,7 @@ AI 编程工具将会话数据存储在跨平台位置。大多数工具在所
 | Qwen CLI | `~/.qwen/` | `%USERPROFILE%\.qwen\` | 所有平台使用相同路径 |
 | Roo Code | `~/.config/Code/User/globalStorage/rooveterinaryinc.roo-cline/tasks/` | `%USERPROFILE%\.config\Code\User\globalStorage\rooveterinaryinc.roo-cline\tasks\` | VS Code globalStorage 任务日志 |
 | Kilo | `~/.config/Code/User/globalStorage/kilocode.kilo-code/tasks/` | `%USERPROFILE%\.config\Code\User\globalStorage\kilocode.kilo-code\tasks\` | VS Code globalStorage 任务日志 |
+| Mux | `~/.mux/sessions/` | `%USERPROFILE%\.mux\sessions\\` | 所有平台相同路径 |
```
[the remark this answers is not in the document]
Comment (junhoyeo) [reply README.zh-cn.md]: Fixed in a58cd9f — removed the extra trailing backslash in the Mux Windows path.
